index.bs

```diff
@@ -487,8 +486,82 @@ Issue(98): Having getFile methods in both FileSystemDirectoryHandle and FileSyst
 with very different behavior might be confusing? Perhaps rename at least one of them (but see also
 previous issue).
 
-Issue(47): Should getEntries be its own method, or should FileSystemDirectoryHandle just be an async
-iterable itself?
+### Directory iteration ### {#api-filesystemdirectoryhandle-asynciterable}
+
+<div class="note domintro">
+  : for await (let [|name|, |handle|] of |directoryHandle|) {}
+  : for await (let [|name|, |handle|] of |directoryHandle| . entries()) {}
+  : for await (let |handle| of |directoryHandle| . values()) {}
+  : for await (let |name| of |directoryHandle| . keys()) {}
+  :: Iterates over all entries whose parent is the entry represented by |directoryHandle|. Entries
+     that are created or deleted while the iteration is in progress might or might not be included.
+     No guarantees are given either way.
+</div>
+
+Advisement: In Chrome this is currently implemented as a `directoryHandle.getEntries()` method that can be used in a `for await..of` loop.
```

Comment:
Would we fix this in Chrome 84? If so, should we call this out?

Reply:
We should definitely update this to mention the milestone chrome implements this, when we know what milestone that is. Not sure currently if that is going to 84 branching next week, but I suspect not. So leaving it like this for now.